packages/parser/src/components/parser/statement/ifStatement.ts

```diff
@@ -36,7 +36,14 @@ export default class IfStatement extends Statement {
 
     const consequent = Statement.getStatementImpl(this._tokenExecutor.getLookahead()!).getStatement();
 
-    const alternate = this._tokenExecutor.getLookahead() != null && this._tokenExecutor.getLookahead()!.type === TokenTypes.WARNA_BHAI ? this._tokenExecutor.eatTokenAndForwardLookahead(TokenTypes.WARNA_BHAI) && Statement.getStatementImpl(this._tokenExecutor.getLookahead()!).getStatement() : this._nullLiteral.getLiteral();
+    let alternate;
+    if (this._tokenExecutor.getLookahead() != null && this._tokenExecutor.getLookahead()!.type === TokenTypes.WARNA_BHAI) {
+      alternate = this._tokenExecutor.eatTokenAndForwardLookahead(TokenTypes.WARNA_BHAI) && Statement.getStatementImpl(this._tokenExecutor.getLookahead()!).getStatement();
+    } else if (this._tokenExecutor.getLookahead() != null && this._tokenExecutor.getLookahead()!.type === TokenTypes.NAHI_TO_BHAI) {
```

Comment:
Instead of handling else if condition altogether in a separate file. We should handle it right here by creating an array of `alternates` and run a loop till we get `warna bhai`  or nothing.

Reply:
Makes sense to change `alternate` to `alternates` now 👍

Reply:
This is done